R/cec.R

```diff
@@ -6,23 +6,31 @@
 #' @param A_K_CEC (numeric) The occupation of the CEC with K (\%)
 #' @param A_CA_CEC (numeric) The occupation of the CEC with Mg (\%)
 #' @param A_MG_CEC (numeric) The occupation of the CEC with Ca (\%)
-#' @param advice (character) Optional parameter to select CEC index for soil structure or fertility. Options: fertility_index or structure_index
+#' @param A_OS_GV (numeric) The organic matter content of soil in percentage
+#' @param B_BT_AK (character) The type of soil
+#' @param advice (character) Optional parameter to select CEC index for soil aggregation or cation buffer capacity. Options: cationbuffer_index or aggregation_index
 #'    
 #' @import data.table
 #' 
 #' @export
-calc_cec <- function(A_CEC_CO,A_K_CEC,A_CA_CEC,A_MG_CEC, advice) {
+calc_cec <- function(A_CEC_CO,A_K_CEC,A_CA_CEC,A_MG_CEC, A_OS_GV, B_BT_AK, advice) {
```

Comment:
The function name is not very descriptive yet

Reply:
calc_cec is a function to quantify two cec dependent functions: aggregate stability and a measure for soil fertility. Is it better to splilt up intwo?

Reply:
function will be split up.